modules/ibmz-configure-nbde-with-static-ip.adoc

```diff
@@ -0,0 +1,145 @@
+Enabling NBDE disk encryption in an {ibmzProductName} or {linuxoneProductName} environment requires additional steps, which are described in detail in this section.
+
+.Prerequisites
+
+* You set up the External Tang Server. See link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/security_hardening/configuring-automated-unlocking-of-encrypted-volumes-using-policy-based-decryption_security-hardening#network-bound-disk-encryption_configuring-automated-unlocking-of-encrypted-volumes-using-policy-based-decryption[Network-bound disk encryption] for instructions.
+* You have installed the `butane` utility.
+* You have reviewed the instructions for how to create machine configs with Butane.
+
+.Procedure
+
+. Create Butane config files for the control plane and compute nodes.
++
+The following example Butane configuration for a control plane node creates a file named `master-storage.bu` for disk encryption:
++
+[source,yaml]
+----
+variant: openshift
+version: 4.13.0
+metadata:
+  name: master-storage
+  labels:
+    machineconfiguration.openshift.io/role: master
+storage:
+  filesystems:
+    - device: /dev/mapper/root
+      format: xfs
+      label: root
+      wipe_filesystem: true
+  luks:
+    - clevis:
+        tang:
+          - thumbprint: QcPr_NHFJammnRCA3fFMVdNBwjs
+            url: http://12.23.21.58:7500
+ifndef::ibm-z-kvm[]
+      device: /dev/disk/by-partlabel/root <1>
+endif::ibm-z-kvm[]
+ifdef::ibm-z-kvm[]
+      device: /dev/disk/by-partlabel/root
+endif::ibm-z-kvm[]
+      label: luks-root
+      name: root
+      wipe_volume: true
+openshift:
+  fips: true
+----
+ifndef::ibm-z-kvm[]
+<1> For installations on DASD-type disks, replace with `device: /dev/disk/by-label/root`.
+endif::ibm-z-kvm[]
+
+. Create a customized initramfs file, by running the following command:
++
+[source,terminal]
+----
+$ coreos-installer pxe customize \
+    /root/rhcos-bootfiles/rhcos-<release>-live-initramfs.s390x.img \
+    --dest-device /dev/sda --dest-karg-append \
+    ip=<ip-address>::<gateway-ip>:<subnet-mask>::<network-device>:none \
+    --dest-karg-append nameserver=<nameserver-ip> \
+    --dest-karg-append rd.neednet=1 -o \
+    /root/rhcos-bootfiles/<Node-name>-initramfs.s390x.img
+----
+ifndef::ibm-z-kvm[]
++
+[NOTE]
+====
+Before first boot, you must customize the initramfs for each node in the cluster and add PXE kernel parameters.
+====
+endif::ibm-z-kvm[]
+
+. Create a parameter file that includes `ignition.platform.id=metal` and `ignition.firstboot`.
++
+Example kernel parameter file for the control plane machine:
++
+ifndef::ibm-z-kvm[]
+[source,terminal]
+----
+rd.neednet=1 \
+console=ttysclp0 \
+coreos.inst.install_dev=/dev/dasda \ <1>
+ignition.firstboot ignition.platform.id=metal \
+coreos.live.rootfs_url=http://10.19.17.25/redhat/ocp/rhcos-413.86.202302201445-0/rhcos-413.86.202302201445-0-live-rootfs.s390x.img \ coreos.inst.ignition_url=http://bastion.ocp-cluster1.example.com:8080/ignition/master.ign \
```

Comment:
Was a line break intended after the first backslash?

Reply:
Yes an extra blank caused the line not to break. Removed that.